composer.json

```diff
@@ -17,7 +18,8 @@
         }
     ],
     "require": {
-        "php": "^8.0"
+        "php": "^8.1",
```

Comment:
If we bump the PHP version, that's likely a new major version release for the library -- any chance of keeping it down at 8.0, even if it means losing some language niceties?